stdlib/3.4/asyncio/events.pyi

```diff
@@ -109,7 +109,9 @@ class AbstractEventLoop(metaclass=ABCMeta):
     @coroutine
     def create_server(self, protocol_factory: _ProtocolFactory, host: str = ..., port: int = ..., *,
```

Comment:
"Changed in version 3.4.4: The host parameter can now be a sequence of strings". Looks like we're missing this part. Would you mind changing the type for host to `Union[str, Sequence[str]]`? We don't care about 3.4.0 through 3.4.3.